app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -18,10 +19,8 @@ type ParamsType = {
   slug: string[];
 };
 
-// Control what happens when a dynamic segment is visited that was not generated with generateStaticParams.
-export const dynamicParams = false;
-export const dynamic = 'force-static';
-export const revalidate = 0;
+export const dynamicParams = false; // Don't generate pages at runtime.
+export const revalidate = 60 * 60 * 24 * 365; // Cache fetch data for the year.
```

Comment:
This appeared to be helping the Safari back button bug but I moved a few other things so I have to revalidate (see what I did there) that this change is doing the trick.